tests/load/pipeline/test_replace_disposition.py

```diff
@@ -57,15 +57,6 @@ def load_items():
     # we should have all items loaded
     table_counts = load_table_counts(pipeline, *[t["name"] for t in pipeline.default_schema.data_tables()])
 
-    # in the classic truncate and insert replace for now no one knows what will happen, just check that we have
-    # max the items that we expect
-    if replace_strategy == "truncate-and-insert":
```

Comment:
amazing!

Reply:
:)

Reply:
I did like the part where it says "no one knows what will happen" it's a shame we remove this golden bit of commenting.